packages/clients/finance-info-client/FinanceInfoClient.js

```diff
@@ -0,0 +1,177 @@
+const { get } = require('lodash')
+const fetch = require('node-fetch')
+
+const conf = require('@open-condo/config')
+const { getRedisClient } = require('@open-condo/keystone/redis')
+
+const { apiToken: API_KEY } = conf['ADDRESS_SUGGESTIONS_CONFIG'] ? JSON.parse(conf['ADDRESS_SUGGESTIONS_CONFIG']) : {}
```

Comment:
Direct queries to dadata? Why do not use the address service?

Reply:
Do address service supports such queries as 
```
/findById/party
/findById/bank
```